website/src/app/pricing/_page.tsx

```diff
@@ -371,8 +371,8 @@ export default function _Page() {
             <p>When will I be billed?</p>
           </blockquote>
           <p className="mb-8">
-            When you start service, or at the beginning of each billing cycle.
-            Enterprise plans are billed quarterly or annually.
+            The Team plan is billed when you start service, or at the beginning
+            of each billing cycle. Enterprise plans are billed annually.
```

Comment:
One question that comes to my mind is, "is the first cycle prorated if I start part way through a cycle?".  Also wondering if "billing cycle" is defined anywhere?